src/components/card/card.tsx

```diff
@@ -1,5 +1,7 @@
 import {OfferListItem} from '../../types';
 import clsx from 'clsx';
+import {Link} from 'react-router-dom';
+import {AppRoute} from '../../constants';
 
 function Card(props: OfferListItem): JSX.Element {
```

Comment:
Лучше делать сразу деструктуризацию в пропсах
```suggestion
function Card({ id, type, ......}: OfferListItem): JSX.Element {
```